### ISSUE TITLE
What are the pre-computed features in 'features.pt'?

### ISSUE THREAD

**Issue by rppravin** [haiciyang/Feature-predictor-for-speech-codec#6]
Thanks for the code.

In dataset_orig.py, looks like some pre-computed features are assumed to be stored in 'features.pt'. Can you please let me know what these features are? Are these Cepstral features computed from Librispeech data using a different code?

Thanks in advance!

**Comment by haiciyang** [comment]
Hi! Yes the features are cepstral features of the speech signal from the training dataset. They were exacted the same way as LPCNet does. LPCNet's repo (https://github.com/xiph/LPCNet) has provided the code. You can refer to the second step of Training a new model - generate training data ([(https://github.com/xiph/LPCNet))](https://github.com/xiph/LPCNet?tab=readme-ov-file#training-a-new-model)

**Comment by rppravin** [comment]
Thank you! Can you also confirm the input features: [0 to 17] - Bark Cepstrum, [18] - pitch lag, [19] - pitch correlation. Is this correct?

**Comment by haiciyang** [comment]
Yes. That's correct. The same with LPCNet's original feature design.
